tests/FakeItEasy.Tests.Approval/ApprovedApi/FakeItEasy/net40.approved.txt

```diff
@@ -198,9 +198,7 @@ namespace FakeItEasy
     [System.Runtime.CompilerServices.NullableAttribute(0)]
     public class static FakeObjectCallExtensions
     {
-        [return: System.Diagnostics.CodeAnalysis.MaybeNullAttribute()]

```

Comment:
It's annoying that we're losing the annotations, because it means we won't see changes to the nullability attributes... But the alternative is to expose a copy of the attributes that could conflict with an existing one.
There's a new version of PublicApiGenerator (released just hours ago!) that adds support for nullable ref types, maybe it does the right thing? Worth trying, I think.